docs/testplan/HA-SmartSwitch-test-plan.md

```diff
@@ -0,0 +1,171 @@
+
+![physical-connection](./Img/topo_smartswitch_ha.png)
+
+Please refer to [sonic-mgmt testbed overview](https://github.com/sonic-net/sonic-mgmt/blob/master/docs/testbed/README.testbed.Overview.md) for more about testbed setup and diagrams. 
+
+
+## Test Methodology
+
+1. Open flow rules are configured on OVS Bridges to control the traffic flow:
+    * All packets sent out from Neighbor are only forwarded to DUT
+    * All packets sent out from DUT are forwarded to Neighbor and PTF
+    * All packets sent out from PTF are only forwarded to DUT
+1. Traffic will be sent from PTF docker, and sniffed on PTF docker by sonic-mgmt test scripts.
+1. Depending on the test cases, switchovers will be triggered and verified. 
+1. Control plane status, DPU counters, metering data will be measured for the tests.
+
+## Test Plan
+Assuming there is a pair of DPUs in the system, and at the step of test setup, we have DPU-1 as active, DPU-2 as standby. 
+
+
+### Downstream (T2->T0) Traffic Verification 
+
+For some of the cases in downstream traffic verification, you will see two versions, one is traffic sending through DPU1, and the other one is traffic sending through DPU2. The purpose of  it is to verify:
+* Traffic shouldn’t be disrupted when failures happen on standby side.
+* Traffic should be tunneled if landing on standby side. 
+
+The name convention of a test case will be “\<Test Scenario\>-[Active|Standby]”, indicating the traffic is sent through the initial active or standby side. 
+
+####  Module 1 	Normal OP and Planned Events
+| Case                      | Goal                                       | Test Steps                                                                                 | Expected Control Plane Behavior                               | Expected Data Plane Behavior                      |
+|---------------------------|--------------------------------------------|--------------------------------------------------------------------------------------------|--------------------------------------------------------------|---------------------------------------------------|
+| Normal OP – Active        | Verify normal operation in healthy state  | • Start downstream I/O through Active side                                                 | DPU1 remains active, DPU2 remains standby.                   | T0 receives packets without disruption.          |
+| Normal OP – Standby       | Verify normal operation in healthy state  | • Start downstream I/O through Standby side                                                | DPU1 remains active, DPU2 remains standby.                   | T0 receives packets without disruption.          |
```

Comment:
for standby, we should still see traffic coming from the active side.

Reply:
addressed